hypha/apply/projects/views/project.py

```diff
@@ -717,23 +702,33 @@ def get(self, request, *args, **kwargs):
             raise
         return response
 
-    def get_paf_data_with_field(self, project):
-        data_dict = {}
-        form_data_dict = project.form_data
-        for field in project.form_fields.raw_data:
-            if field['id'] in form_data_dict.keys():
-                if isinstance(field['value'], dict) and 'field_label' in field['value']:
-                    data_dict[field['value']['field_label']] = form_data_dict[field['id']]
-
-        return data_dict
-
-    def get_supporting_documents(self, project):
-        documents_dict = {}
-        for packet_file in project.packet_files.all():
-            documents_dict[packet_file.title] = self.request.build_absolute_uri(
-                reverse('apply:projects:document', kwargs={'pk': project.id, 'file_pk': packet_file.id})
-            )
-        return documents_dict
+
+@method_decorator(staff_or_finance_or_contracting_required, name='dispatch')
+class ProjectDetailDocxView(SingleObjectMixin, View):
+    model = Project
+
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()
+
+        project = self.get_object()
+
+        context = get_paf_download_context(self.request, project)
+
+        context['show_footer'] = False
+
+        template_path = 'application_projects/paf_export.html'
+        template = get_template(template_path)
+        html = template.render(context)
+
+        buf = io.BytesIO()
+        document = Document()
+        new_parser = HtmlToDocx()
+        new_parser.add_html_to_document(html, document)
+        document.save(buf)
+
+        response = HttpResponse(buf.getvalue(), content_type='application/docx')
+        response['Content-Disposition'] = f'attachment; filename="{project.title}.docx"'
```

Comment:
The filename here should be url encoded, and it would be nice to have a `date` appended to it as well. See below: 

```
20220810-project-title.pdf
```

Reply:
Added the date as a prefix and slugify the filename.